eng/pipelines/docindex.yml

```diff
@@ -15,33 +15,59 @@ jobs:
           echo "##vso[task.setvariable variable=docfxPath;isOutput=true]$(Build.SourcesDirectory)/docfx/docfx.exe"
```

Comment:
One thing to keep in the back of your mind when we move the doc index stuff to eng/common we should also consider sharing this yml file or at least some level of template from this file.

Reply:
Sure. I have checked the yaml file. Only the template folder path looks different, but we plan to move all to common folder. So it is doable to move the docindex.yml into common.